packages/calendar/src/Calendar.ts

```diff
@@ -89,8 +89,8 @@ export class Calendar extends SpectrumElement {
     @property({ type: Boolean, reflect: true })
     padded = false;
 
-    @state()
-    private currentDate!: CalendarDate;
+    @property({ attribute: false })
+    currentDate!: CalendarDate;
```

Comment:
I see this is used in tests. Are there other use cases? If not, we should likely move it back to `private` and work around that reality in tests, rather than surfacing a property that we don't need to the public API.